src/wizard/SetupWizard.php

```diff
@@ -78,7 +80,7 @@ public function run() : bool{
 			}
 		}while($lang === null);
 
-		$this->lang = new Language($lang);
+        $this->lang = new Language($lang, LanguageManager::loadLang(RESOURCE_PATH . 'locale/', $lang), new Language(LanguageManager::FALLBACK_LANGUAGE, LanguageManager::loadLang(RESOURCE_PATH . 'locale/', LanguageManager::FALLBACK_LANGUAGE)));
```

Comment:
Is it too hard to keep the code formatting consistent with the project your editing?